server/env.go

```diff
@@ -0,0 +1,27 @@
+package server
```

Comment:
So did `--custom-links` just create new routes which were read by the ui, thus not have to export any direct info to the frontend?

Reply:
Custom links already had a route. If they were provided they got added to `/chronograf/v1/`

https://github.com/influxdata/chronograf/blob/feature/env/server/routes.go#L49